packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -941,6 +943,16 @@ mixin WidgetInspectorService {
     _errorsSinceReload = 0;
   }
 
+  bool isStructuredErrorsEnabled() {
+    return const bool.fromEnvironment('structuredErrors');
```

Comment:
drive by nit: I would give this some sort of namespace so it is less likely to conflict with a user define: flutter.inspector.structuredErrors